internal/uvm/types.go

```diff
@@ -124,9 +123,11 @@ type UtilityVM struct {
 	// The CreateOpts used to create this uvm. These can be either of type
 	// uvm.OptionsLCOW or uvm.OptionsWCOW
 	createOpts interface{}
+
 	// Network config proxy client. If nil then this wasn't requested and the
 	// uvms network will be configured locally.
-	ncProxyClient ncproxyttrpc.NetworkConfigProxyService
+	// ncProxyClient ncproxyttrpc.NetworkConfigProxyService
```

Comment:
Accidental commenting out of the above that was removed. The new field needs a comment describing the address now also